cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -321,6 +322,17 @@ func (c *CStorVolumeReplicaController) cVRAddEventHandler(
 
 	// IsEmptyStatus is to check if initial status of cVR object is empty.
 	if IsEmptyStatus(cVR) || IsInitStatus(cVR) || IsRecreateStatus(cVR) {
+		// We should generate replicaID for new volume only
+		if IsEmptyStatus(cVR) || IsInitStatus(cVR) {
+			if err := volumereplica.GenerateReplicaID(cVR); err != nil {
+				klog.Errorf("cVR ReplicaID creation failure: %v", err.Error())
+				return string(apis.CVRStatusOffline), err
```

Comment:
status being set to 'Offline' doesn't reconcile for volume creation. This issue is with current code.
We need to fix these things in CVR cleanup. @mittachaitu can you please create issue for this in github for HacktoberFest?